backend/yugioh/management/commands/seed.py

```diff
@@ -0,0 +1,240 @@
+import logging
+
+import requests
+from card.models import Card
+from django.core.management.base import BaseCommand
+from game.models import Game
+from psycopg2 import DataError
+from yugioh.models import YugiohCard, YugiohCardRarity, YugiohCardSet, YugiohCardInSet
+
+log_filename = "yugioh_seeding.log"
+logging.basicConfig(filename=log_filename, level=logging.INFO,
```

Comment:
No need to write the logs to a file. I think it's better to log them to the output, so they can be immediately seen, also captured by the docker logs in the near future.